### DISCUSSION TITLE
case insensitive organizeImports

### DISCUSSION THREAD

**Discussion by eMerzh** [biomejs/biome#2162] [Ideas]
Hello :) 

I know we can add options like this all day long,
but this seems kinda simple, so i try 😬 

We love the auto organise import (in eslint then in biome)
But in biome, we can't really say that we want to sort regardless of the case.

it would be a nice little addition :)

**Comment by Conaclos** [comment]
Why do you want case-insensitive sort?

**Reply by eMerzh** [reply]
ahah ofc... why not :p 
 because in my mind 
import {Fragment, lazy } from 'react' could be sorted alphabetically

**Reply by Conaclos** [reply]
I see. I think it is not case insensitivity that you want (otherwise you get a partial order between letters), but rather a different total order between letters. An order in which `A` and `a` could be before `b` and `B`, and so on. However, `A` could still come before `a` (or the vice versa) in order to ensure a total order between letters.

**Reply by nikhilgupta345** [reply]
Would love this as well

**Reply by Conaclos** [reply]
This was implemented in Biome 2.0. Moreover, we now have an option to control the order between specifiers: we support `natural` (the default order) and [lexicographic](https://biomejs.dev/assist/actions/organize-imports/#change-the-sorting-of-import-identifiers-to-lexicographic-sorting).
